style.css

```diff
@@ -1,713 +1,817 @@
-  width: 40px;
-  height: 40px;
-  border-radius: 8px;
-  padding-right: 8px;
-}
-
-.web-lang {
-  display: flex;
-  flex-direction: row;
-  height: 50.72px;
-  width: 303.5px;
-  margin-top: 0;
-  padding-top: 5px;
-  background-color: #f7f7f9;
-}
-
-.web-lang img {
-  margin-left: 40px;
-  width: 48px;
-  height: 48px;
-}
-
-.web-lang span {
-  margin-left: 20px;
-  padding-top: 15px;
-
-  /* padding-top: -30px; */
-}
-
-.web-lang li {
-  margin-left: 16px;
-  margin-top: 18px;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-size: 15px;
-  line-height: 20px;
-  color: #253858;
-}
-
-ul.socail-media-icon1 li {
-  display: inline-block;
+*,
+*::before,
+*::after {
   margin: 0;
   padding: 0;
-  list-style: none;
-}
-
-hr {
-  color: #dfe1e6;
-  width: 340px;
-  height: 0.2px;
-}
-
-.contact-me {
-  margin: 0 auto;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  padding-top: 70px;
-  padding-right: 10px;
-  border-top-left-radius: 60px;
-  background-color: #6070ff;
-  width: 330px;
-  margin-top: 200px;
-
-  /* margin-left: 10px; */
-  background-image: url("./Image/contact\ form\ shapes-mobile@2x.png");
-  background-repeat: no-repeat;
-  background-position: right;
-  background-size: 230px 730px;
-  background-position-x: 119px;
-  background-position-y: 20px;
-}
-
-.contact-title {
-  font-family: 'Poppins', sans-serif;
-  font-size: 40px;
-  color: #fff;
-  text-align: center;
-}
-
-.contact-text {
-  color: #ebebff;
-  font-family: 'Poppins', sans-serif;
-  font-size: 20px;
-  text-align: center;
-  margin-left: 24px;
-  margin-right: 24px;
-  margin-top: -27px;
-  margin-bottom: 20px;
-}
-
-.form-container {
-  margin-left: 25px;
-}
-
-.first-name {
-  color: #172b4d;
-  padding: 10px 16px;
-  width: 260px;
-  border-radius: 8px;
-  border-width: 0;
-  margin-top: 20px;
-}
-
-.contact-textarea {
-  color: #b3bac5;
-  margin-top: 18px;
-  border-radius: 8px;
-  border-width: 0;
-  padding: 10px 16px;
-  resize: none;
-}
-
-.git-in-touch {
-  color: #6070ff;
-  margin-top: 20px;
-  margin-bottom: 45px;
-  border-radius: 8px;
-  border-width: 0;
-  padding: 8px 12px;
-  font-size: 17px;
-  font-weight: 500;
-  font-family: 'Poppins', sans-serif;
-}
-
-.git-in-touch:hover {
-  background-color: #2230d2;
-  color: #fff;
-  margin-top: 20px;
-  margin-bottom: 45px;
-  border-radius: 8px;
-  border-width: 0;
-  padding: 8px 12px;
-  font-size: 17px;
-  font-weight: 500;
-  font-family: 'Poppins', sans-serif;
+  box-sizing: border-box;
 }
 
-.git-in-touch :active {
-  background-color: #070c44;
-  color: #fff;
-  margin-top: 20px;
-  margin-bottom: 45px;
-  border-radius: 8px;
-  border-width: 0;
-  padding: 12px 16px;
-  font-size: 17px;
-  font-weight: 500;
-  font-family: 'Poppins', sans-serif;
+body {
+  background-color: #e5e5e5;
+  font-family: "Poppins", sans-serif;
+  overflow-x: hidden;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+@media screen and (min-width: 320px) {
+  .container {
```

Comment:
- [ ]  I noticed here that you are not following the `mobile-first `approach. The `mobile-first` approach is about designing for mobile devices first and then using media queries to add styles for larger screens. But currently, you have added a media query for the mobile designs. I will suggest that you leave the mobile styles as they were before without a media query. The media query is only needed for the desktop styles since we are practicing the `mobile-first` approach. So kindly remove this media query :100: 
- [ ] While doing this, please ensure that you don't mess up your styles so your design looks great on both mobile and desktop as I mentioned earlier :wink: